app/urls.py

```diff
@@ -18,5 +18,6 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', include('jobseeker.urls')),
+    path('', include('feed.urls')),
+    path('jobseeker/', include('jobseeker.urls')),
```

Comment:
@paOmer What do you think about this **Suggestion:**

Edit the line of:

```py
path('jobseeker/', include('jobseeker.urls')),
```
to be:
```py
path('', include('jobseeker.urls')),
```

So the URL path to the "About" page, would be like so:

![image](https://user-images.githubusercontent.com/70590583/158504279-23cf3fdc-c8ed-437b-b6e3-463337244faa.png)

Reply:
I was thinking about that, but am not sure if this is a legit way of doing so because the feed app also have an empty url path to make it the main page.

Hi @Yarboa @kasemAlem,  is there a right way of doing so?

Reply:
Instead of using css, recommended to use boostrap
https://getbootstrap.com/docs/5.0/getting-started/introduction/

We will focus it on UI talk within 10 days

Reply:
Working on model only from now.
I am using bootstrap for the website design and just applying minor adjustments to it with my own css file.